flink-connectors/flink-connector-hive/src/test/resources/query-test/udf.q

```diff
@@ -0,0 +1,13 @@
+-- SORT_QUERY_RESULTS
+
+select bround(55.0, -1);
```

Comment:
Is there any test which the second parameter is positive?

Reply:
I add a query `select bround(55.0, +1)` to cover the positive number with `+` sign.
For the positive number with `+` sign, I think the query `select percentile_approx(x, 0.5) from foo` in `udaf.q` has covered it.